src/main/scala/dx/core/languages/wdl/ParseSource.scala

```diff
@@ -53,15 +54,14 @@ case class ParseSource(logger: Logger) {
 
   private def bInfoFromDoc(doc: TAT.Document): BInfo = {
     // Add source and adjuncts for main file
-    val (sources, adjunctFiles) = doc.sourceUrl match {
-      case None => throw new RuntimeException("Document URL required")
-      case Some(url) if url.getProtocol == "file" =>
-        val absPath: Path = WdlUtil.getLocalPath(url)
+    val (sources, adjunctFiles) = doc.source match {
+      case localFs: LocalFileSource =>
+        val absPath: Path = localFs.localPath
         val sources = Map(absPath.toString -> doc)
         val adjunctFiles = Adjuncts.findAdjunctFiles(absPath)
         (sources, adjunctFiles)
-      case Some(url) =>
-        val sources = Map(url.toString -> doc)
+      case fs =>
```

Comment:
What is `fs`?

Reply:
abbreviation for FileSource - I changed it to be more clear